ubuntu22.04/nvidia-driver

```diff
@@ -15,6 +15,8 @@ NVIDIA_UVM_MODULE_PARAMS=()
 NVIDIA_MODESET_MODULE_PARAMS=()
 NVIDIA_PEERMEM_MODULE_PARAMS=()
 TARGETARCH=${TARGETARCH:?"Missing TARGETARCH env"}
+
+OPEN_KERNEL_MODULES_ENABLED=${OPEN_KERNEL_MODULES_ENABLED:-""}
```

Comment:
Does this statement lead to this variable being set at all times?

Reply:
It will be set to the empty string by default. I can update this if required. I have confirmed though that `[ -n ${OPEN_KERNEL_MODULES_ENABLED} ]` return true if the variable is unset or is the empty string.

Reply:
Thanks